examples/frequency.py

```diff
@@ -0,0 +1,48 @@
+
+import multidict as multidict
+
+import numpy as np
+import time
+
+import re
+from PIL import Image
+from os import path
+from wordcloud import WordCloud
+import matplotlib.pyplot as plt
+
+def getFrequencyDictForText(sentence):
+    fullTermsDict = multidict.MultiDict()
+    tmpDict = {}
+
+    # making dict for counting frequencies
+    for text in sentence.split(" "):
+        if re.match("a|the|an|the|to|in|for|of|or|by|with|is|on|that|be",text):
+            continue
+        val = tmpDict.get(text,0)
+        tmpDict[text.lower()] = val+1
+    for key in tmpDict:
+        fullTermsDict.add(key,tmpDict[key])
+    return fullTermsDict
+
+
+
+def makeImage(text):
+    alice_mask = np.array(Image.open("alice_mask.png"))
+
+    stopwords = {" "}
```

Comment:
the stopwords are actually ignored when using ``generate_from_frequencies`` so this is unnecessary.

Reply:
My gosh..
I didn't know that it can work without stopwords. And I fix promptly as fast as I can.